build/commands/lib/config.js

```diff
@@ -93,7 +93,7 @@ const Config = function () {
   this.defaultGClientFile = path.join(this.rootDir, '.gclient')
   this.gClientFile = process.env.BRAVE_GCLIENT_FILE || this.defaultGClientFile
   this.gClientVerbose = getNPMConfig(['gclient_verbose']) || false
-  this.targetArch = getNPMConfig(['target_arch']) || 'x64'
+  this.targetArch = getNPMConfig(['target_arch']) || process.arch
```

Comment:
this will not work correctly because nodejs process.arch does not line up with expectations for gn target_cpu. In particular `x32` vs `x86`

Reply:
do we need `x86` somewhere?

Reply:
see what you get if you search for `ia32`

Reply:
https://nodejs.org/api/process.html#processarch
```
The operating system CPU architecture for which the Node.js binary was compiled. Possible values are: 'arm', 'arm64', 'ia32', 'mips','mipsel', 'ppc', 'ppc64', 's390', 's390x', 'x32', and 'x64'.
```